src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -495,29 +495,27 @@ exports[`calltree/ProfileCallTreeView computes a width for a call tree of a real
     >
```

Comment:
All the changes here come from the move to enzyme. This is quite straightforward: looks like enzyme retains the "key" props in the output, and otherwise this is some space changes.